File: fstp-python-2.7/src/com/purefun/fstp/core/bo/model/MarketBarBO.py
import uuid

class MarketBarBO(object):

    def __init__(self):
        self.product_id = ''
        self.exch = ''
        self.date = ''
        self.high = 0.0
        self.low = 0.0
        self.open = 0.0
        self.close = 0.0
        self.change = 0.0
        self.volume = 0.0
        self.uuid = str(uuid.uuid1())
        self.boid = 10
        self.destination = "fstp.md.bar"

File: fstp-core-bo/src/main/java/com/purefun/fstp/core/bo/MarketBarBO.java
package com.purefun.fstp.core.bo;

import com.purefun.fstp.core.bo.tool.fstpbo;

@fstpbo(boid = 10L, destination = "fstp.md.bar")
public class MarketBarBO extends BaseBO{
	public String product_id = "";
	public String exch = "";
	public String date = "";
	public double high = 0.0;
	public double low = 0.0;
	public double open = 0.0;	
	public double close = 0.0;
	public double change = 0.0;
	public double volume = 0.0;
}
